tests/unit/parsers/html-google-sheets-test.js

```diff
@@ -0,0 +1,77 @@
+import HTMLParser from 'mobiledoc-kit/parsers/html';
+import PostNodeBuilder from 'mobiledoc-kit/models/post-node-builder';
+import Helpers from '../../test-helpers';
+import GoogleSheets from '../../fixtures/google-sheets';
+import { forEach } from 'mobiledoc-kit/utils/array-utils';
+import { CARD_TYPE } from 'mobiledoc-kit/models/types';
+
+const {module, test} = Helpers;
+
+function parseHTML(html, options={}) {
+  let builder = new PostNodeBuilder();
+  return new HTMLParser(builder, options).parse(html);
+}
+
+module('Unit: Parser: HTMLParser Google Sheets');
```

Comment:
I'd prefer to see these without the big fixtures file. The best for of testing would also not be to test against another parsed string, but against a post. For example:

```js
test('#parse returns a markup section when given single line of text', (assert) => {
  let text = 'some text';
  let post = parser.parse(text);
  let expected = Helpers.postAbstract.build(({post, markupSection, marker}) => {
    return post([markupSection('p', [marker('some text')])]);
  });

  assert.postIsSimilar(post, expected);
});
```

Taken from https://github.com/bustlelabs/mobiledoc-kit/blob/e003067ec9a49ceebd7520b5c785f91136e3261c/tests/unit/parsers/text-test.js#L26-L34

This testing is more exact, only the actual values are parsed. The expected value is just an abstract.